Nodejs/Product/Nodejs/SharedProject/ProjectNode.cs

```diff
@@ -6018,23 +6025,21 @@ private void SetupProjectGlobalPropertiesThatAllProjectSystemsMustSet()
         /// </remarks>
         private bool TryBeginBuild(bool designTime, bool requiresUIThread = false)
         {
-            IVsBuildManagerAccessor accessor = null;
-
-            if (this.Site != null)
+            if (this.Site != null && this.buildManagerAccessor == null)
             {
-                accessor = this.Site.GetService(typeof(SVsBuildManagerAccessor)) as IVsBuildManagerAccessor;
+                this.buildManagerAccessor = this.Site.GetService(typeof(SVsBuildManagerAccessor)) as IVsBuildManagerAccessor;
```

Comment:
>this.Site.GetService(typeof(SVsBuildManagerAccessor)) as IVsBuildManagerAccessor [](start = 44, length = 80)

What would it mean for this to succeed here if it didn't work in the ctor?  Can that happen?